en/firefox/whatsnew/whatsnew-fx79.ftl

```diff
@@ -0,0 +1,49 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/79.0/whatsnew/all/
+
+# HTML page title
+whatsnew79-page-title = What’s new with { -brand-name-firefox }
+
+# Main title
+whatsnew79-main-title = Keep your data private from everyone – including us
+whatsnew79-main-intro = The internet should be a safe place for all. That’s why { -brand-name-firefox } doesn’t follow you around online, and stops others from following you, too.
+
+# Main CTA button
+whatsnew79-primary-cta = Make { -brand-name-firefox } Default
+
+# Fallback CTA button, shown if Firefox is already a user's default browser
+whatsnew79-fallback-cta = Get the { -brand-name-firefox } App
+
+whatsnew79-secure-your-connection = Secure your connection
+whatsnew79-with-one-tap-mozilla-vpn = With one tap, { -brand-name-mozilla-vpn } encrypts your connection to keep your online activity private on any network — safe from hackers and other prying eyes.
```

Comment:
need to add Mozilla VPN to the brands.ftl.

Reply:
oh right, I did that but forgot to include it in the PR 🤦